### PR TITLE
fix: (Core) make Form's Input Message readable by screen readers

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/4229
#### Please provide a brief summary of this pull request.
Form Message become readable by screen readers like VoiceOver
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [n/a] Stackblitz works for all examples

### CODE REVIEW

**Review by Betrozov — APPROVED** [review]

**Review by mykolavorobiov — APPROVED** [review]

**Review by sKudum — APPROVED** [review]

**Review by kavya-b — APPROVED** [review]

File: libs/core/src/lib/form/form-message/form-message.component.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ export type MessageStates = 'success' | 'error' | 'warning' | 'information';
     selector: 'fd-form-message',
     templateUrl: './form-message.component.html',
     styleUrls: ['./form-message.component.scss'],
+    host: {
+        '[attr.aria-live]': '"assertive"',
+        '[attr.aria-atomic]': 'true'
```
Comment (dimamarksman) [inline libs/core/src/lib/form/form-message/form-message.component.ts:25]: can we skip binding here?

Comment (dimamarksman) [inline libs/core/src/lib/form/form-message/form-message.component.ts:25]:
```suggestion
        'aria-live': 'assertive',
        'aria-atomic': 'true'
```

**Review by dimamarksman — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by InnaAtanasova** [comment]
e2e tests are failing

**Review by JKMarkowski — APPROVED** [review]
@olshansky 
I think focusin / focusout is much better than just a basic clicks. Could you set then `closeOnEscapeKey/closeOnOutsideClick` to false? Otherwise it's going to be reversed (opened in focusout, closed on focusin)

**Pull request merged**